tensorflow_toolkit/text_detection/README.md

```diff
@@ -157,6 +157,7 @@ python tools/test.py \
       --input_model model/weights/export/frozen_graph.pb \
       --reverse_input_channels \
       --data_type FP32 \
+      --input_shape="[1,768,1280,3]" \
```

Comment:
@AlexanderDokuchaev could you modify CI script so that this parameter is added to `export` section of CI job

Reply:
Changed.
Can i merge now?

Reply:
I think yes, thank you